pypesto/objective/petab_import.py

```diff
@@ -76,7 +76,8 @@ def from_folder(folder,
 
         return PetabImporter(
             petab_problem=petab_problem,
-            output_folder=output_folder)
+            output_folder=output_folder,
+            model_name=model_name)
```

Comment:
Just assuming: This is due to api-changes in petab?

Reply:
correct.